src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -140,15 +141,17 @@ def is_bicep_file(file_path):
 
 def get_bicep_available_release_tags():
     try:
-        response = requests.get("https://aka.ms/BicepReleases")
+        ca_file = certifi.where()
+        response = requests.get("https://aka.ms/BicepReleases", verify=ca_file)
```

Comment:
This make `requests` only use `certifi.where()`. Env var `REQUESTS_CA_BUNDLE` and `CURL_CA_BUNDLE` no longer takes effect.